src/ruptures/costs/costnormal.py

```diff
@@ -12,16 +13,31 @@ class CostNormal(BaseCost):
 
     model = "normal"
 
-    def __init__(self):
-        """Initialize the object."""
+    def __init__(self, add_small_diag=True):
+        """Initialize the object.
+
+        Args:
+            add_small_diag (bool, optional): For signals with truly constant
+                segments, the covariance matrix is badly conditioned, so we add
+                a small diagonal matrix. Defaults to False.
```

Comment:
```suggestion
                a small diagonal matrix. Defaults to True.
```

Reply:
Thanks, solved in 43e6ffb1409792a41bf8b0169c25782a906448f1